packages/components/src/components/tables/es-table/es-table.tsx

```diff
@@ -183,12 +183,19 @@ export class Table {
                         })}
                     >
                         {this.loading &&
-                            cell.loading?.(h, {
-                                ...(this.extraCellProps?.(key, data) ?? {}),
-                                key,
-                                data,
-                                parent: this.identifier,
-                            })}
+                            cell.width !== '0' &&
```

Comment:
Having a 0 width cell is very unusual (and was basically a hack on the organizations overview to fit two data shapes on a single table), I don't think it's worth having a special case for it here, when we can just add `loading: () => null` in that one place.